scaffolds.json

```diff
@@ -16,5 +16,19 @@
     "repo": "https://github.com/onflow/scaffold-flow.git",
     "description": "Simple demo application using next.js and FCL with provided Cadence contracts.",
     "commit": "4158c5e11b6d2c9e1cbc99af597e3b42b2e6c193"
+  },
+  {
+    "name": "Simple Unity",
+    "repo": "https://github.com/onflow/UnityFlowSDK.git",
+    "description": "Simple examples how to interact with Flow network using Unity SDK.",
```

Comment:
Could be: `"Simple example demonstrating how to interact with the Flow network using Unity SDK."`